### DISCUSSION TITLE
Nerf de l'attaque sombre en général

### DISCUSSION THREAD

**Discussion by romain22222** [Crownicles/Crownicles#2524] [Suggestions fermées]
Bonsoir, je viens proposer un nerf de l'attaque sombre. En effet, elle est si puissante qu'elle fait à elle seule toute la puissance de la classe mage. Elle est si puissante que c'est la seule qui peut faire plus de 500 dégâts à niveau égal. Elle est si puissante qu'elle fait réhausser artificiellement le niveau des boss sur l'île.

Ma proposition : une seule attaque sombre par combat autorisé, à l'instar de la bénédiction.

**Comment by romain22222** [comment]
Rectification sur le niveau égal : à -20 niveaux par rapport à l'adversaire

**Comment by Voisin1234** [comment]
Bonjour ☺️

**Comment by Voisin1234** [comment]
Alors je ne sais pas pour l'île mais pour ce qui concerne notre combat, si ma sombre a fait plus de 500 c'est pas parce que tu étais concentré?

**Reply by romain22222** [reply]
maybe, mais même, l'attaque fait trop de trucs positifs sans retour négatif (-15% atk adverse, scalling sur l'attaque adverse, hauts dégâts)

**Reply by Voisin1234** [reply]
C’est sûr, cela fait longtemps que je n’ai pas joué avec cette classe. Je crois qu’il y a déjà eu beaucoup de débats à ce sujet. Je ne suis pas assez informé, mais je pense que limiter l’attaque à une seule fois par combat n’est peut-être pas la meilleure idée. Déjà qu’elle ne fonctionne pas tout le temps, si en plus elle échoue la seule fois où elle est utilisée, ce ne serait pas terrible.

**Reply by romain22222** [reply]
La bénédiction échoue aussi, faudrait pour les équilibrer qu'elles ne ratent pas quand utilisés normalement (ie en gros pas quand l'adversaire est intouchable)

**Comment by Jajadu69** [comment]
Bonjour,  si on met la sombre comme la bénédiction.  Soit une par fight , quel est le point fort de la classe mage ? 
Pas de possibilité de se regen, pas d'attaque faisant réellement des dégâts mis à part la sombre du coup, qui loupe quand même assez souvent.  

De plus elle ne fait jamais 500 dégâts,  sauf contre l'araignée quand elle fait la discrétion ou les vétérans après leur concentration.

**Reply by Voisin1234** [reply]
Et qui, en soi, la bénédiction n'est pas la meilleure des attaques, car tu dois attendre au maximum avant de l'utiliser pour avoir des meilleurs statistiques, donc vers la fin du combat, et en plus elle fait très peu de dégâts (Bon après c'est un autre débat.)

**Reply by Jajadu69** [reply]
Ça voudrait dire faire exactement pareil ? Plus on attend plus on fait de dégâts .
Et la bene loupe beaucoup également.

**Reply by LaMangousteRose** [reply]
![Screenshot_2024-07-07-15-02-22-301_com android chrome](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/154462445/d10c72af-8473-4646-986b-027f5380b304)

**Reply by LaMangousteRose** [reply]
De rien 😉 pas plus tard que ce matin

**Reply by Cirpacha** [reply]
Oui c'est ce que jaja a dit, les 500 dégâts c'est seulement avec le boost de dégâts du vétéran...
Hésite surtout pas à argumenter tes 6 réactions "👎", là ça fait rageux sans arguments

**Reply by LaMangousteRose** [reply]
La provocation je m'y connais

**Reply by Procion06** [reply]
![0](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/107723160/a41df01b-ef7b-4645-8858-3ba89b91ddc5)

**Reply by Procion06** [reply]
pas plus tard que cet aprèm

**Comment by Ravenclaw-l** [comment]
Demander à ce que la Sombre ressemble à la béné est probablement la pire suggestion, surtout au vu de l'état actuel du chevalier en PvP. La sombre est assez capricieuse, elle n'a pas besoin d'être nerf plus que ça.

**Comment by OrionYp** [comment]
Je pense qu'il faut juste supprimer le boost de dégâts qu'à la sombre après une concentration d'un vétéran /d'une araignée

**Comment by Procion06** [comment]
Encore un joueur qui ne connaît pas bien la classe et exagère grandement
la sombre est capée à 380 en crit (sauf boost de dégâts qui s'applique dans ce cas-là aux 2 combattants et donc à l'attaque de son adversaire) et fait moins de dégâts que d'autres attaques qui coûtent moins de souffle !
dans ce cas il faut être honnête et appliquer la restriction que tu proposes à toutes les classes qui possèdent des attaques faisant autant sinon plus de dégâts...
Et que dire de la simple ou de la rapide qui fait presque jeu égal avec la sombre en crit pour seulement 2 et 3 de souffle
A noter aussi que dans les 2 lignes de ton post tu utilises 3 fois les mots 'si puissante" comme le ferait un publicitaire pour vendre son produit xD
Je rappelle que ce qui est affirmé sans preuve peut être rejeté sans preuve 
En l'état :
La sombre est si nulle que de nombreuses attaques font beaucoup plus de dégâts pour un coût moindre en souffle
La sombre est si nulle qu'elle rate très régulièrement
La sombre est si nulle que vous pouvez perdre un combat contre un joueur qui a 15 levels de moins

Re-Nerf **inutilement** la sombre reviendrait juste à faire disparaître le mage de la méta
et par conséquent à appauvrir la diversité des combats
on manque déjà cruellement de joueurs ayant envie de fight, là ce serait encore pire

**Comment by StrangerIM** [comment]
La sombre est bien comme elle est

**Comment by Carsebel** [comment]
Bonjour,
Je propose simplement d'accepter la défaite et d'arrêter de pleurer. La sombre se saborde toute seule, aucune utilité de la nerf et de la mettre comme la bénédiction (qui fonctionne 1 fois sur 15 pour certains cheva ^^)
Donc arrêtez de pleurer et soyez fair play...
On dirait des caprices de gosses

**Comment by QQtin3** [comment]
Je vois pas l'utilité des propos de certains, il serait peut être envisageable de donner votre opinion sans critiquer les autres ? Merci beaucoup.
Je ne vais pas donner mon avis car je ne joue presque plus mais faites un peu preuve de gentillesse dans vos propos et soyez constructifs...

Concernant le cap à 380 de damage, j'imagine que romain ne sort pas cette stat de nulle part, vous pouvez en parler entre vous pour comprendre d'où ça vient.

**Reply by Cirpacha** [reply]
380 est pas un nombre démesuré au vu des chances que l'attaque rate et du souffle requis. Personnellement je vois pas le problème, la sombre est sûrement la seule attaque du mage qui se nerf toute seule lol, pas besoin d'en rajouter.
D'autres suggestions plus utiles pour nerf le mage auraient pu être de réduire la baisse des stats de l'adversaire de la souffle/sombre ou d'augmenter le souffle nécessaire pour faire un vol de souffle 🤷. En tout cas la proposition d'une seule sombre par combat semble assez nulle, pour quoi faire ? Ce n'est pas une attaque divine. De plus toutes les classes ont une attaque qui fait des dégâts, que ce soit l'intense avec son crit à 350, la chargée (crit : 370), la lourde (crit: 350), la puissante (crit: 340), l'ultime (crit: 700), ou encore la divine (SANS crit: 360-400). Les nombres sont ceux que j'ai trouvés vite fait dans les combats de l'offi, me tombez pas dessus. Mais on voit bien que nerf la sombre n'est pas pertinent du tout, de nombreuses autres attaques font à peu près les mêmes dégâts...

**Reply by romain22222** [reply]
je confirme que toutes les classes ont une attaque qui fait beaucoup de dégâts, mais : 
- la chargée demande un tour de chargement, donc pas utilisable tous les tours loin de là
- la lourde est quasiment capée à ces dégâts, elle n'a pas de cas extrême lui permettant d'abuser de certaines situations.
- l'ultime / la divine est à utilisation unique par nature et leur powercreep est présent en conséquence.
- les autres j'ai pas sufisamment joué contre, je peux pas donner mon avis

De plus, le mage a un truc qui est supérieur aux autres classes : des dégâts continus : c'est la seule classe qui permet de faire des dégâts supplémentaires chaque tour. Ca a l'air de rien, mais ca veut dire qu'elle peut sortir des 200/300 plus souvent que n'importe quelle autre classe si on se met pas soi même en altération d'état. Pas toutes les classes le peuvent, et celles qui le peuvent, soit c'est stupide (étourdi), soit ca revient à perdre un tour sur 2, ce que le mage peut donc compenser avec un mélange de vol de souffle et de... sombre, ce qui lui permet d'avoir sur toutes les classes un moyen d'arriver à ses fins : au choix elle emet des altés à gogo et abuse de son adversaire, ou l'empêche de jouer, ou abuse de l'attaque de son adversaire.

**Reply by Voisin1234** [reply]
Question à part, quel est le taux de réussite de l'attaque lourde ?

**Reply by Cirpacha** [reply]
Eh ben, faut croire que tu as sacrément envie de râler aujourd'hui...
Bon, pour la chargée, le "tour de chargement" fait des dégâts non négligeables, ce n'est pas un tour de chargement.
La lourde est selon toi capée à 350, c'est déjà pas mal hein, en plus elle enlève 25% de la défense de l'adversaire y a pas de quoi la rabaisser ici, c'est une attaque puissante. (même si son taux de réussite n'est pas très élevé, tout comme la sombre d'ailleurs)
L'ultime est à utilisation unique mais en cas de crit à 700 ça fait 350 dégâts par tour en 2 tours donc heureusement que l'utilisation est unique, surtout qu'elle ne coûte "que" 10 souffles.
La divine n'est pas à utilisation unique, 2 sont permises par combat, soit 400*2=800 dégâts en 2 tours donc le pala qui cumule ultime et divines : 700+800=1500 dégâts, oh surprise c'est beaucoup...

Ensuite en ce qui concerne les dégâts continus je suppose que tu parles de la poison. Alors oui le mage a cette spécificité car toutes les classes sont uniques et possèdent des atouts que les autres n'ont pas. Par contre tu dérives bien rapidement de la sombre, pourquoi ne pas avoir proposé de nerf la poison dès le début si c'est ce que tu voulais ?
Enfin peu importe, il est utile de rappeler que le canonnier a l'attaque boomerang, qui inflige des dégâts sur plusieurs tours également, et pas des moindres. Donc même là il faut pouvoir se mettre une alté, et comme tu le dis "pas toutes les classes le peuvent" donc le mage ne peut pas se protéger contre l'attaque boomerang. Pour autant, est-ce si grave ? 

Non, chaque classe a ses avantages et inconvénients, c'est ce qui fait leur diversité et qui permet une variété des combats, indispensable pour ne pas s'ennuyer.
Le tank riposte les attaques, le cano envoie des boomerangs et fait de nombreux dégâts avec le combo canons, le pala stun et utilise des attaques très puissantes en fin de combat, le vete se regen, le mage empoisonne et baisse les stats de ses adversaires (et le cheva rate ses attaques) et c'est génial comme ça, on va pas tout modifier parce que telle ou telle classe a des attaques que les autres n'ont pas, c'est précisément le but d'avoir plusieurs classes à disposition.

**Comment by kirit000** [comment]
Je pose ça là. L'attaque canon qui est peu coûteuse en souffle et avec une grande chance de faire une critique après une deuxième utilisation fait autant de dégâts qu'une attaque sombre

**Comment by kirit000** [comment]
Sachant que ces deux dernieres ont déjà été nerf donc ça s'arrête là.

**Comment by BastLast** [comment]
Étant donné qu'il vous est impossible de débattre en adulte, cette suggestion est classée comme refusée et ce thread fermé a tout nouveau message, merci de vous comporter comme des adultes dans les futures suggestions où il en sera de même. Je tiens a préciser qu'il est possible de ban sur github également si nécessaire et que je n'hésiterais pas a le faire si nécessaire.

**Reply by BastLast** [reply]
Concernant cet équilibrage, etant donné la toxicité de cet espace commentaire, il ne sera pas tenu compte des retours fait sur cette suggestion, à l'avenir, comportez vous correctement.
